modules/instrument_builder/jsx/react.app.js

```diff
@@ -0,0 +1,485 @@
+TabPane = React.createClass({
+    render: function() {
+        var classList = "tab-pane";
+        if(this.props.Active) {
+            classList += " active"
+        }
+        return (
+            <div className={classList} id={this.props.TabId}>
+                <h1>{this.props.Title}</h1>
+                <br />
+                {this.props.children}
+            </div>
+            );
+    }
+});
+LoadPane = React.createClass({
+	getInitialState: function() {
+	 	return {
+	 		alert: ''
+	 	};
+	},
+	chooseFile: function (e) {
+		var value = e.target.files[0];
+		this.setState(function(state){
+			return {
+				file: value,
+				alert: ''
+			};
+		});
+	},
+	setAlert: function (type) {
+		this.setState(function(state){
+			return {
+				alert: type
+			};
+		});
+	},
+	resetAlert: function () {
+		this.setState(function(state){
+			return {
+				alert: ''
+			};
+		});
+	},
+	loadFile: function () {
+		var callback = {
+				success: this.props.loadCallback,
+				error: this.setAlert
+			};
+		Instrument.load(this.state.file, callback);
+	},
+	render: function () {
+		var spanDownStyle = {
+				display: 'none'
+			},
```

Comment:
This variable doesn't seem to be used (and if it is, there's no reason for it.. if something shouldn't be displayed, just don't render it in the jsx instead of hiding it with CSS)